tools/depends/target/libdvdnav/LIBDVDNAV-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=libdvdnav
 BASE_URL=https://github.com/xbmc/libdvdnav
-VERSION=6.0.0-Leia-Alpha-3
+VERSION=6.1.1-Next-Nexus-Alpha2
 ARCHIVE=$(LIBNAME)-$(VERSION).tar.gz
-SHA512=11c93eaacd156f8fd7dec7c43d366438b201f31ad55b2870463a9e286912b6ada08882319a021fb7992190f87b909a49f2b83e0321cc17aedc29f7fe5898fa72
+SHA512=079edcd7732b4edbf90806d9acc0a03efef386612df48fa711fd944320034630ad89dbdb7d72bdea0560b7dac71229057c02cb8bf0c9d70e1b86680bfd08b84d
```

Comment:
there seems to be a hash mismatch, calculates
`01c415a561ea8bd9d568ad2668fbf8bc54304557e413776e1b841145cad3f7b9991a6772af7bac408b2cb87c6236c378e79c1c023a58b280a6c92fee151f0fb1` for me

Reply:
yeah, was hoping the hash would be the same after pushing to the new repo. Fixed